src/data/contributors/Raviteja.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Please delete the rest of the files only change your json file, Also, please  add your github handle.

Reply:
okay, I created a new pull request for that